services-directions/src/main/java/com/mapbox/api/directions/v5/models/BannerView.java

```diff
@@ -34,7 +34,7 @@ public static Builder builder() {
   }
 
   /**
-   * Plain text with all the {@link BannerComponents} text combined.
+   * Plain text with all the {@link BannerComponents} text combined.BannerInstructions.
```

Comment:
Typo?

Reply:
Yes. https://github.com/mapbox/mapbox-java/pull/1111 will address this.